website/docs/docs/supported-data-platforms.md

```diff
@@ -3,81 +3,102 @@ title: "Supported Data Platforms"
+| DuckDB                 | [dbt-duckdb](https://github.com/jwills/dbt-duckdb/)                                             | [DuckDB Profile](duckdb-profile)           |                                            | [#db-duckdb](https://getdbt.slack.com/archives/C039D1J1LA2)       | Community     |
+| Dremio                 | [dbt-dremio](https://github.com/fabrice-etanchaud/dbt-dremio/)                                  | [Dremio Profile](dremio-profile)           |                                            | n/a                                                               | Community     |
+| Exasol Analytics       | [dbt-exasol](https://github.com/tglunde/dbt-exasol)                                             | [Exasol Profile](exasol-profile)           |                                            | n/a                                                               | Community     |
+| Firebolt               | [dbt-firebolt](https://github.com/firebolt-db/dbt-firebolt)                                     | [Firebolt Profile](firebolt-profile)       | [Firebolt Configs](firebolt-configs)       | [#db-firebolt](https://getdbt.slack.com/archives/C02PYT5CXN0)     | Firebolt      |
+| AWS Glue               | [dbt-glue](https://github.com/aws-samples/dbt-glue)                                             | [Glue Profile](glue-profile)               | [Glue Configs](glue-configs)               | [#db-glue](https://getdbt.slack.com/archives/C02R4HSMBAT)         | AWS           |
+| Greenplum              | [dbt-greenplum](https://github.com/markporoshin/dbt-greenplum)                                  | [Greenplum Profile](greenplum-profile)     | [Greenplum Configs](greenplum-configs)     | n/a                                                               | Community     |
+| Hive                   | [dbt-hive](https://github.com/cloudera/dbt-hive)                                                | [Hive Profile](hive-profile)               | [Hive Configs](hive-configs)               | [#db-hive](https://getdbt.slack.com/archives/C0401DTNSKW)         | Cloudera      |
+| Impala                 | [dbt-impala](https://github.com/cloudera/dbt-impala)                                            | [Impala Profile](impala-profile)           | [Impala Configs](impala-configs)           | [#db-impala](https://getdbt.slack.com/archives/C03K2PTHHTP)       | Cloudera      |
+| iomete                 | [dbt-iomete](https://github.com/iomete/dbt-iomete)                                              | [iomete Profile](iomete-profile)           |                                            | [#db-iomete](https://getdbt.slack.com/archives/C03JFG22EP9)       | Iomete        |
+| Layer Bigquery         | [dbt-layer](https://github.com/layerai/dbt-layer)                                               | [Layer Profile](layer-profile)             |                                            | [#tools-layer](https://getdbt.slack.com/archives/C03STA39TFE)     | Layer AI      |
```

Comment:
> Layer Bigquery

Might have to just call this Layer? BigQuery is Google's trademark product name, don't think Layer can use it like this.

Reply:
It's also a single repo/plugin that now supports both BigQuery + Snowflake (https://github.com/layerai/dbt-layer/pull/125)

Reply:
ah didn't notice that @jtcohen6 -- i'll ask them to update their profile docs page then
```suggestion
| Layer         | [dbt-layer](https://github.com/layerai/dbt-layer)                                               | [Layer Profile](layer-profile)             |                                            | [#tools-layer](https://getdbt.slack.com/archives/C03STA39TFE)     | Layer AI      |
```